ISM43362/ISM43362.cpp

```diff
@@ -196,6 +195,76 @@ bool ISM43362::dhcp(bool enabled)
     return (_parser.send("C4=%d", enabled ? 1 : 0) && check_response());
 }
 
+/*
+  * @brief  Update wifi_module_country_code.
+  * @param  country_code: User's country code.
+  * @retval boolean value : false ->error
+
+WiFi module is only able to receive 4 country codes: CA, US, FR and JP.
+CA, US -> 11 channels
+FR     -> 13 channels
+JP     -> 14 channels
+
+Thus 3 country code groups can be defined :
+
+- Country Code Group corresponding to 11 channels : CountryCodeElevenChannels,
+- Country Code Group corresponding to 13 channels : CountryCodeThirteenChannels,
+- Country Code Group corresponding to 14 channels : CountryCodeFourteenChannels.
+
+For an user's country code (input country code), the function is specifying
+to which Country Code Group it is belonging.
+
+For instance,
+user's country code=VN (Vietnam - 13 channels) is belonging
+to CountryCodeThirteenChannels.
+
+Thus function is specifying that WIFI module must be configured
+with country code = FR.
+
+Function is updating WIFI_module_country_code with FR.
+
+*/
+
+#ifdef MBED_CONF_ISM43362_WIFI_COUNTRY_CODE
+bool ISM43362::check_country_code(const char* country_code)
+{
```

Comment:
in mbed-os coding style the { are located at the end of line
```
bool ISM43362::check_country_code(const char* country_code) {
```
applies to all lines, maybe A-style can help

Reply:
It seems that the coding style ({ at the end of line) is almost never used in the entire WiFi module. Must I make the modification in all files ?

Reply:
hmm ok, forget about my comment for now then. If the coding style is already not consistent you may fix that in a separate Pull Request.